pom.xml

```diff
@@ -97,29 +97,4 @@
     </plugins>
   </build>
 
-  <reporting>
```

Comment:
I'm confused, why removing all that if we enable then in the pipeline ?

Reply:
The pipeline uses its own libs.